cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database")
```

Comment:
It is a good practice to use logging instead of print statements or stdout.write. It provides more flexibility by allowing you to set the level of messages you want to log and is helpful in debugging.